### PR TITLE
Use text area fields for name, description and notes

### PR DESCRIPTION
**Pull request by frwickst**
Fixes https://github.com/City-of-Helsinki/ltj/issues/54

### CODE REVIEW

File: nature/forms.py [hunk 1]
```diff
@@ -3,6 +3,21 @@
...
+        widget=forms.Textarea,
+        max_length=80,
+        required=False,
+    )
+    description = forms.CharField(
+        widget=forms.Textarea,
+        max_length=255,
+        required=False,
+    )
+    notes = forms.CharField(
```
Comment (mingfeng) [inline nature/forms.py]: Redefine the fields means the duplication of model field information (required, max_length), I think it is simpler to specify the widgets for the fields in Meta class if that's the only thing changed.

File: nature/forms.py [hunk 2]
```diff
@@ -3,6 +3,21 @@
 
 
 class FeatureForm(forms.ModelForm):
+    name = forms.CharField(
+        widget=forms.Textarea,
+        max_length=80,
```
Comment (mingfeng) [inline nature/forms.py]: Make name a text field might be misleading as it only allow 80 chars, would it be better to make the input field a bit longer?

**Comment by frwickst** [comment]
@mingfeng Done, you are indeed correct that this is a lot nice, don't know why I did not hit me to use the widgets, still a bit rusty regarding the django admin I guess, thank for the comments 👍

**Comment by frwickst** [comment]
This is strange, how is Flake8 complaining, I've fixed the problem that is complains about and all looks good locally...

**Pull request merged**